packages/edge_runtime/lib/src/utils.dart

```diff
@@ -1,25 +1,12 @@
 import 'dart:typed_data' show ByteBuffer, Uint8List;
 
 Object? convertBody(Object? body) {
-  if (body == null) {
-    return null;
-  }
-
-  if (body is String) {
-    return body;
-  }
-
-  if (body is Uint8List) {
-    return body;
-  }
-
-  if (body is ByteBuffer) {
-    return body;
-  }
-
-  throw ArgumentError.value(
-    body,
-    'body',
-    'Body must be an nullable instance of [String], [ByteBuffer] or [Uint8List].',
-  );
+  return switch (body) {
+    String || Uint8List || ByteBuffer || null => body,
```

Comment:
That won't work. You're not performing type checks, but rather `body == String`

Instead do:

```suggestion
    String() || Uint8List() || ByteBuffer() || null => body,
```

Reply:
Oh right nice catch brain got messed up choosing the String() or String _ syntax !